content/OTP/Specifications/OTP_validation_protocol.adoc

```diff
@@ -6,9 +6,13 @@ All requests are HTTP GET requests. As such, all parameters must be
 properly URL encoded.  In particular, some base64 characters (such as
 "+") in the value fields needs to be escaped.
 
+Each request sent to the server can be signed. To ensure that the
+server rejects any requests that have been tampered with, clients
+should sign each request they make to the server.
+
 Each response sent by the server is signed.  To verify that the
-response has not been tampered with, clients either verify the HMAC
-signature or use HTTPS connections (and verify the server
+response has not been tampered with, clients should verify the HMAC
+signature and must use HTTPS connections (and verify the server
 certificate).
 
 === Generating signatures
```

Comment:
Maybe add an example of input parameters and resulting signed message?

Reply:
How about I add some to the "Test Vectors" page?

Reply:
Sounds good! Might be worth adding a link from here too, then.